core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -97,7 +97,7 @@ var TopicEditorPage = function() {
   var topicThumbnailUploadInput = element(
     by.css('.protractor-test-photo-upload-input'));
   var topicThumbnailCropper = element(
-    by.css('.protractor-test-photo-crop'));
+    by.css('.cropper-container'));
```

Comment:
I think we should avoid using class names that is used for styling for testing as well.
Using a `.protractor-test-cropper-container` and adding the class to the element would be better?

Reply:
Hi @jameesjohn, the "cropper-container" class is part of a div that is dynamically generated by the cropper library. This is why I did not use a `protractor-*` class in this e2e test.